benchexec/result.py

```diff
@@ -133,26 +133,39 @@ class Property(collections.namedtuple("Property", "filename is_svcomp name")):
 
     __slots__ = ()  # reduce per-instance memory consumption
 
-    def compute_score(
-        self, category, result, witness_category=WITNESS_CATEGORY_UNKNOWN
-    ):
+    def _adjust_score_witness(score, witness_category):
```

Comment:
A "score witness" is not an existing concept, the method would better be named `adjust_score_for_witness_validation`.

Reply:
Yes, this is what I had in mind, and omitted 'for' and 'validation'.  Will change.

Reply:
Done.